src/Apps/Order/Routes/Shipping2/Components/FulfillmentDetails.tsx

```diff
@@ -91,45 +91,6 @@ export const FulfillmentDetails: FC<FulfillmentDetailsProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [hasSavedAddresses])
 
-  /*
-   * Re-save fulfillment details on load if they are already saved &
-   * require artsy shipping (new address form mode only - saved addresses
-   * handle this separately)
-   */
-  useEffect(() => {
-    const { savedFulfillmentDetails } = shippingContext.orderData
-
-    const isArtsyShippingSaved =
-      savedFulfillmentDetails?.fulfillmentType === FulfillmentType.SHIP &&
-      savedFulfillmentDetails.isArtsyShipping &&
-      shippingContext.meData.addressList.length === 0
-
-    if (isArtsyShippingSaved) {
-      const refreshShippingQuotes = async () => {
-        // instead of handleSubmit, call the save fulfillment details function
-        // directly
-        const result = await handleSaveFulfillmentDetails({
-          attributes: savedFulfillmentDetails.attributes,
-          fulfillmentType: FulfillmentType.SHIP,
-          meta: {
-            // FIXME: Will clobber previous address verification (but we can't
-            // know what the previous status was until we can read from server)
-            addressVerifiedBy: null,
-          },
-        })
-
-        shippingContext.actions.setIsPerformingOperation(false)
-
-        if (result) {
-          shippingContext.actions.setStage("shipping_quotes")
-        }
-      }
-
-      refreshShippingQuotes()
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [])
-
```

Comment:
@erikdstock - you know this code best, but I would disagree with this change. This stuff seems to be co-locatable here, given that it concerns fulfillment details. If we move it to the parent (which simply coordinates sub-dependencies) we're bloating the parent with details that operationally concern fulfillment.

Reply:
I hear your point and that's why it started out there, but if you view this as a list of steps to be completed, this action actually spans these two steps - it's really about automatically advancing to the next one.

That said I'm ok with reverting this one change because it doesn't affect the functionality.